assets/js/pam/components/card/ShipActivityCardComponent.vue

```diff
@@ -8,7 +8,7 @@
           <span>Total</span>
           <div class="total-values">
             <span class="dayInSea text-blue-france text-bold">{{ activite.nb_jours_mer }} jours en mer</span>
-            <span class="missionTrackTime text-blue-france text-bold">52 heures de mission</span>
+            <span class="missionTrackTime text-blue-france text-bold">{{ total }} heures de mission</span>
```

Comment:
Total ou calculTotal ?

Reply:
`total` présent dans les data

Reply:
OK compris. Je pense que tu peux peut-être utiliser un computed et que ce sera plus simple pour total. Je valide et Merge comme ça

Reply:
Noté pour les prochaines 🆗